client/src/components/DataFiles/DataFilesModals/DataFilesUploadModal.js

```diff
@@ -49,8 +49,14 @@ const DataFilesUploadModal = ({ className, layout }) => {
         }
       });
   };
-  const disabled =
+  const dropZoneDisabled =
     Object.values(status).filter(s => s === 'UPLOADING').length > 0;
+  const uploadButtonDisabled =
+    dropZoneDisabled ||
+    (!dropZoneDisabled &&
+      uploadedFiles.length ===
```

Comment:
I somehow was able to get into a state where the `Upload` button was enabled with a rejected file and a successful file. If I can't figure out how to reproduce, I'll resolve this comment

<img width="1087" alt="Screen Shot 2021-05-28 at 11 15 03 AM" src="https://user-images.githubusercontent.com/20326896/120014536-cf1a8f00-bfa7-11eb-9ab1-fd18ada69b70.png">

Reply:
I can't reproduce it either, I'll keep testing.
<img width="1081" alt="Screen Shot 2021-05-28 at 12 22 30 PM" src="https://user-images.githubusercontent.com/62672123/120020613-95e61d00-bfaf-11eb-860c-50bcd10bd844.png">

Reply:
Me neither - could've been a local quirk. I'll resolve for now!